includes/Admin.php

```diff
@@ -1283,7 +1356,7 @@ public function process_order_document_form_data( $form_data, $document_slug )
 			$minute       = ! empty( $form_data['_wcpdf_'.$document_slug.'_date']['minute'] ) ? $form_data['_wcpdf_'.$document_slug.'_date']['minute'] : '00';
 
 			// clean & sanitize input
-			$date         = date( 'Y-m-d', strtotime( $date ) );
+			$date         = gmdate( 'Y-m-d', strtotime( $date ) );
```

Comment:
Yes, I decided to make the change based on the Plugin Check error below:

>ERROR: WordPress.DateTime.RestrictedFunctions.date_date  
The `date()` function is affected by runtime timezone changes, which can lead to incorrect date/time displays. It is recommended to use `gmdate()` instead.

WordPress suggests using `gmdate()` to avoid these issues, as it ensures consistency and aligns with the timezone settings configured under *Settings > General > Timezone*.

In my opinion, our current implementation is more problematic. By using `date()`, we risk displaying dates that do not accurately reflect the timezone set in WordPress, which could lead to inconsistencies for users. Furthermore, the server's timezone might be incorrectly set, or the server itself could be located in a different timezone from the store, making it difficult for the store owner to adjust the date accurately. However, let me know your opinion on this.

Reply:
I think you're right. I did some test while calling `get_last_date()`. My local server is set to `UTC` and my WordPress timezone setting is set to Sydney. I switched the timezone to New York during the call. These were the results for both `date()` and `gmdate()`:

```
Local server timezone: UTC
Local server time: 2024-12-11 12:30:06
WP set to: Australia/Sydney

Last date with local server timezone set to: UTC
Local time (date()): 2024-12-11 23:30:06
GMT time (gmdate()): 2024-12-11 23:30:06

Last date with local server timezone set to: America/New_York
Local time (date()): 2024-12-11 18:30:06
GMT time (gmdate()): 2024-12-11 23:30:06
```